frontend/amundsen_application/static/js/components/DefinitionList/index.tsx

```diff
@@ -35,7 +35,14 @@ export const DefinitionList: React.FC<DefinitionListProps> = ({
         </dt>
         <dd className="definition-list-definition">
           {typeof description === 'string' ? (
-            <SanitizedHTML html={description} />
+            <SanitizedHTML
```

Comment:
we don't usually do inline styling, and it seems you added these to a class on css as well so can you use that instead?